### ISSUE TITLE
OneStop time filter investigation

### ISSUE THREAD

**Issue by dneufeldcu** [cedardevs/onestop#1062]
### Summary
<table>
  <tr>
    <td><b>As a</b></td>
    <td>federated search provider</td>
  </tr>
  <tr>
    <td><b>I want to be able to</b></td>
    <td>
       understand the current behavior of the OneStop API results pertaining to my time queries.
    </td>
  </tr>
  <tr>
    <td><b>So that I can</b></td>
    <td>determine whether to proceed with my release that integrates OneStop</td>
  </tr>
</table>

### Tasks

- [ ] Review and document explanation around time filter behavior

**Comment by ajakz** [comment]
I looked into the time range filter being applied against the different collections and discovered: 
1. In most cases the collection had a declared range that went well beyond the query range, so not matching all the files is expected and observed behavior
2. In a number of other cases, the collection had a declared range that did not intersect the query range at all, so not matching any files is expected and observed behavior
3. In the remaining number of cases, the collection had a declared range that did not accurately encompass the granules that existed within it (but a disjoint filter of the same date range found the remaining granules). This is a metadata error.

In short, our time filter works correctly... and there's some inaccurate metadata out there.

The metadata team now knows this additional information and can address the collection/granules time bound discrepancy where it exists. We can continue to work on our documentation to clarify how to use our search api... and perhaps see if we can add any count analytics for collections that show how many granules actually match a collection's  bounds?
